framework/src/org/checkerframework/framework/type/AnnotatedTypeFactory.java

```diff
@@ -366,6 +368,17 @@ public AnnotatedTypeFactory(BaseTypeChecker checker) {
         }
     }
 
+    /**
+     * Issue an error if any of the support qualifiers has @Target that contains something besides
```

Comment:
ErrorReporter.errorAbort(...) does issue an error using the report mechanism and then aborts processing.  I updated the documentation to indicate that this method aborts.